src/datepicker/Datepicker.vue

```diff
@@ -71,6 +72,13 @@ export default defineComponent({
       type: Date as PropType<Date>,
       required: false,
     },
+    /**
+     * Disabled dates for picking
+     */
+    disabledDates: {
+      type: Object as PropType<{ dates: Date[] | undefined }>,
```

Comment:
Is there a reason to make the prop a single-property object instead of just using an array?

Reply:
In the original project [https://github.com/icehaunter/vuejs-datepicker](https://github.com/icehaunter/vuejs-datepicker) we have different options how to provide disabled dates. So we can implement other options in the future without breaking the prop format.

Reply:
Ok, that is a good argument. I would suggest using a shorter optional prop notation though:

```suggestion
      type: Object as PropType<{ dates?: Date[] }>,
```